src/components/common/navbar/NavBar.tsx

```diff
@@ -33,6 +34,12 @@ const NavBar = (): JSX.Element => {
 
         <WithAuthentication AuthorisedComponent={Logout} UnauthorisedComponent={Login} />
       </Header.ActionBar>
+
```

Comment:
This will appear to non authenticated users, make sure it only shows to authenticated uses.

Reply:
Fixed.